wiki/reinstalling-microsoft-store.md

```diff
@@ -0,0 +1,65 @@
+
+    ![](img/reinstalling-microsoft-store/opensettings.png)
+
+2. Select "Apps".
+
+    <img src="img/reinstalling-microsoft-store/apps.png" style="width:300px;">
+
+3. Find "Microsoft Store" and click advanced options.
+
+    <img src="img/reinstalling-microsoft-store/microsoftstore.png" style="width:500px">
+
+> If you can't find the Microsoft Store here, you will have to reinstall it completely. Follow Method 2 to reinstall the Microsoft Store
+
+4. Scroll down and click the reset button. Then wait for a check mark to appear.
+
+    <img src="img/reinstalling-microsoft-store/resetmicrosoftstore.png" style="width:500px">
+
+### Method 2: Resetting the Microsoft Store through WSReset
+
+> The WSReset function is outdated and may not function correctly. If Method 1 of resetting the Microsoft Store cache doesn't work, try reinstalling the Microsoft Store. This method is still available.
+
+1. Open the Run dialog. You can do this by searching for it in the start menu, or by pressing <kbd>Win</kbd> + <kbd>R</kbd>.
+
+    <img src="img/reinstalling-microsoft-store/run.png" style="width:400px">
+
+2. Enter `wsreset` and hit <kbd>Enter</kbd> or press "OK".
+
+    <img src="img/reinstalling-microsoft-store/wsreset.png" style="width:400px">
+
+3. A black window will pop up and when it's finished, restart your computer and it should be fixed.
+
+>If this does not fix your problem, try the Method 2: Reinstalling the Microsoft Store instead.
+
+## Method 2: Reinstalling the Microsoft Store
+
+1. Launch Powershell as administrator by either right-clicking the Windows Start Menu button or by pressing <kbd>Win</kbd> + <kbd>X</kbd> and press "Powershell (Admin)" or by searching it up in the Start Menu then right click and select "Run as Administrator". Either way you will be prompted with a User Account Control Prompt. Hit yes or enter an administrator's password to continue.
+
+    <img src="img/reinstalling-microsoft-store/powershell.png" style="width:600px">
+
+2. Enter the command `Get-AppxPackage *windowsstore* | Remove-AppxPackage` and hit <kbd>Enter</kbd>.
+
+> This command will remove the Microsoft Store and it is irreversible, the nest command will install the Microsoft Store back.
+
+    <img src="img/reinstalling-microsoft-store/removingmicrosoftstore.png" style="width:600px">
+
+3. After the command finishes, enter the command `Get-AppXPackage *WindowsStore* -AllUsers | Foreach {Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppXManifest.xml"}` and hit <kbd>Enter</kbd>.
+
+    <img src="img/reinstalling-microsoft-store/installingmicrosoftstore.png" style="width:600px">
+
+4. After everything your Powershell should look like this. If it looks different or displays some kind of error, check if there are any typing mistakes made. If you are still having issues, it's worth trying some more [general troubleshooting steps](troubleshooting.md), and then walk through this guide again.
```

Comment:
"After completing these steps, Power**S**hell should look like this. If it looks different or displays some kind of error, check you typed the commands correctly"